person.rb

```diff
@@ -0,0 +1,28 @@
+# Class containing data about a person
+require_relative 'nameable'
+
+class Person < nameable
```

Comment:
- [ ] Hello! 👋 Fantastic job on the project so far! 🚀 I've noticed that there's an opportunity to enhance the Person class in accordance with the project's guidelines, which you can find [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md). 📜

- One of the requirements specified in the project guidelines is to include a method called correct_name within the Person class. This method should be designed to return the value stored in the name attribute of the Person instance. 🧐

- Let's make sure to add this method to the Person class to meet the project's expectations and create a well-rounded class! If you have any questions or need further assistance, please feel free to ask. 🤗👍

Reply:
corect name method added.